jcache/pom.xml

```diff
@@ -0,0 +1,141 @@
+<!--
+  ~ JBoss, Home of Professional Open Source
+  ~ Copyright 2011 Red Hat Inc. and/or its affiliates and other
+  ~ contributors as indicated by the @author tags. All rights reserved.
+  ~ See the copyright.txt in the distribution for a full listing of
+  ~ individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <groupId>org.infinispan.quickstart</groupId>
+   <artifactId>jcache-quickstart</artifactId>
+   <version>7.0.0-SNAPSHOT</version>
+
+   <name>Infinispan JCache Quickstart</name>
+   <description>This quickstart demonstrates how to use infinispan as a JCache (JSR 107) provider.</description>
+
+   <url>http://jboss.org/infinispan</url>
+   <licenses>
+      <license>
+         <name>GNU Lesser General Public License</name>
+         <url>http://www.gnu.org/copyleft/lesser.html</url>
+         <distribution>repo</distribution>
+      </license>
+   </licenses>
+
+   <properties>
+      <!-- Explicitly declaring the source encoding eliminates the following
+         message: -->
+      <!-- [WARNING] Using platform encoding (UTF-8 actually) to copy filtered
+         resources, i.e. build is platform dependent! -->
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+      <infinispan.version>7.0.0.Final</infinispan.version>
```

Comment:
Actually, I started with 7.1.0.Final but went back to 7.0.0.Final to align with the other quickstart:

https://github.com/infinispan/infinispan-quickstart/blob/master/embedded-cache/pom.xml#L49

If you want, I can try to update all the quickstart to 7.1.0.Final ? Actually, maybe the ISPN version should lives in the parent of the quickstart, so that updating requires only one file change ?